src/coreclr/vm/corhost.cpp

```diff
@@ -754,15 +751,19 @@ HRESULT CorHost2::CreateDelegate(
 
         if (pMD->HasUnmanagedCallersOnlyAttribute())
         {
+            pMD->PrepareForUseAsAFunctionPointer();
```

Comment:
```suggestion
```
Left-over?

Reply:
I added this since we are giving out a function pointer and thought we should match the behavior in `CEEInfo::getFunctionFixedEntryPoint` and `RuntimeMethodHandle_GetFunctionPointer`. I assumed it was simply missed and was adding it here. Am I mis understanding the utility of this API?